crates/parser/tests/snapshots/statements/valid/0047@1.snap

```diff
@@ -0,0 +1,86 @@
+---
+source: crates/parser/tests/statement_parser_test.rs
+description: CREATE DATABASE x OWNER abc CONNECTION LIMIT 5;
+---
+Parse {
+    cst: SourceFile@0..47
+      CreatedbStmt@0..47
+        Create@0..6 "CREATE"
+        Whitespace@6..7 " "
+        Database@7..15 "DATABASE"
+        Whitespace@15..16 " "
+        Ident@16..17 "x"
+        Whitespace@17..18 " "
+        DefElem@18..27
+          Owner@18..23 "OWNER"
+          Whitespace@23..24 " "
+          Ident@24..27 "abc"
+        Whitespace@27..28 " "
+        DefElem@28..38
+          Connection@28..38 "CONNECTION"
+        Whitespace@38..39 " "
+        Limit@39..44 "LIMIT"
+        Whitespace@44..45 " "
+        Iconst@45..46 "5"
```

Comment:
These should be part of DefElement

Reply:
thanks, fixed in 45f88be683ba728c8454d02c6a2feb1349f3ce58